app/controllers/concerns/restaurants_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RestaurantsController < ApplicationController
+  def index
+    @restaurants = Restaurant.all
+  end
+
+  def show
+    @restaurant = Restaurant.find_by_id(params[:id])
+  end
+
+  def create
```

Comment:
🔨 @Alisher778 the `create` action is responsible for creating resources, so this isn't correct, also this homework only involves the index and show actions, so you can remove